types/events.go

```diff
@@ -55,10 +55,12 @@ type Throttling struct {
 
 type CpuUsage struct {
 	// Units: nanoseconds.
-	Total  uint64   `json:"total,omitempty"`
-	Percpu []uint64 `json:"percpu,omitempty"`
-	Kernel uint64   `json:"kernel"`
-	User   uint64   `json:"user"`
+	Total              uint64   `json:"total,omitempty"`
+	Percpu             []uint64 `json:"percpu,omitempty"`
+	PercpuInKernelmode []uint64 `json:"percpuInKernelmode,omitempty"`
```

Comment:
ditto

Reply:
I've changed names in this `CpuUsage ` struct `PercpuInKernelmode` > `PercpuKernel ` and `PercpuInUsermode` -> `PercpuUser`. These names are better fit to other in this struct.